mysql-test/suite/rocksdb/t/use_direct_reads.test

```diff
@@ -0,0 +1,22 @@
+--source include/have_rocksdb.inc
+
+--echo Checking direct reads
+--let --rocksdb_use_direct_reads=1
```

Comment:
Missing variable name, breaking the testcase?
However it also looks like the testcase would (quite naturally) not work if MTR is run on a non-O_DIRECT FS, e.g. tmpfs if MTR --mem option is given. Please consider, if feasible, skipping the testcase on such systems

Reply:
I think --mem is supported only on non-windows platforms. Maybe adding a have_nomem.inc include file that checks to see if the directory path starts with /dev/shm? It won't catch all cases, but should be sufficient.

Should this line also be "let $_mysqld_option=--rocksdb_use_direct_reads=1"